app/src/main/java/com/rkm/tasky/feature/agenda/screen/6DaySelector.kt

```diff
@@ -0,0 +1,121 @@
+package com.rkm.tasky.feature.agenda.screen
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.rkm.tasky.ui.theme.SixDaySelectorDateTextColor
+import com.rkm.tasky.ui.theme.SixDaySelectorDayCardSelected
+import com.rkm.tasky.ui.theme.SixDaySelectorDayCardUnselected
+import com.rkm.tasky.ui.theme.SixDaySelectorSelectedDay
+import com.rkm.tasky.ui.theme.SixDaySelectorUnselectedDay
+
+@Composable
+internal fun SixDaySelector(
+    modifier: Modifier,
+    days: List<DayUiModel>,
+    onClick: (Long) -> Unit
+) {
+
+    var selectedIndex by remember { mutableStateOf(0) }
```

Comment:
I'd strongly recommend to put all UI states in a ViewModel, since here they won't survive config changes. After a screen rotation or dark/light theme switch, all your text fields will be empty again. With `rememberSaveable` you could work around that, but then your UI will have too many responsibilities, since it also needs to decide how the state is updated.